moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -388,6 +391,8 @@ class KinematicsBase
    * @param tip_frame The tip of the chain
    * @param search_discretization The discretization of the search when the solver steps through the redundancy
    * @return True if initialization was successful, false otherwise
+   *
+   * Note: this is intended to be removed in future ROS versions in favor of robot_model being passed in
```

Comment:
It seems like that shouldn't be the case though - if a deprecated function calls another deprecated function, shouldn't that be OK? Any C++ gurus that know a workaround?